src/xenia/ui/window_win.cc

```diff
@@ -680,9 +680,200 @@ bool Win32Window::HandleMouse(UINT message, WPARAM wParam, LPARAM lParam) {
   return e.is_handled();
 }
 
+static KeyEvent::Key MapVkKeyToKeyEventKey(LPARAM key) {
+  switch (key) {
+    case VK_ESCAPE:
+      return KeyEvent::Key::kEsc;
+    case VK_F1:
+      return KeyEvent::Key::kF1;
+    case VK_F2:
+      return KeyEvent::Key::kF2;
+    case VK_F3:
+      return KeyEvent::Key::kF3;
+    case VK_F4:
+      return KeyEvent::Key::kF4;
+    case VK_F5:
+      return KeyEvent::Key::kF5;
+    case VK_F6:
+      return KeyEvent::Key::kF6;
+    case VK_F7:
+      return KeyEvent::Key::kF7;
+    case VK_F8:
+      return KeyEvent::Key::kF8;
+    case VK_F9:
+      return KeyEvent::Key::kF9;
+    case VK_F10:
+      return KeyEvent::Key::kF10;
+    case VK_F11:
+      return KeyEvent::Key::kF11;
+    case VK_F12:
+      return KeyEvent::Key::kF12;
+    case VK_OEM_3:
+      return KeyEvent::Key::kTick;
+    // 0-9 and A-Z match ASCII
+    case '1':
+      return KeyEvent::Key::k1;
+    case '2':
+      return KeyEvent::Key::k2;
+    case '3':
+      return KeyEvent::Key::k3;
+    case '4':
+      return KeyEvent::Key::k4;
+    case '5':
+      return KeyEvent::Key::k5;
+    case '6':
+      return KeyEvent::Key::k6;
+    case '7':
+      return KeyEvent::Key::k7;
+    case '8':
+      return KeyEvent::Key::k8;
+    case '9':
+      return KeyEvent::Key::k9;
+    case '0':
+      return KeyEvent::Key::k0;
+    case VK_OEM_MINUS:
+      return KeyEvent::Key::kMinus;
+    case VK_OEM_PLUS:
+      return KeyEvent::Key::kEquals;
```

Comment:
Is this correct?

Reply:
Good question. That one confused me too. 

I think it is: 

https://www.gamedev.net/forums/topic/293214-virtual-keycode-hell---or-where-is-vk_equals/